clicommand/agent_start.go

```diff
@@ -756,6 +748,12 @@ var AgentStartCommand = cli.Command{
 			}
 		}
 
+		if cfg.CancelGracePeriod <= cfg.SignalGracePeriodSeconds {
```

Comment:
Would be nice to make the names consistent (and even nicer if they were just `time.Duration` instead of `int`), but I can see how we got here. 🤷

Reply:
Yeah, in next major version, I would want to change both to parse durations. There are already some args that are parsed as durations, but as these are related, and I was constrained by `cancel-grace-period`, I ended up with this.